packages/calling-stateful-client/src/CallAgentDeclarative.ts

```diff
@@ -209,8 +209,20 @@ class ProxyCallAgent implements ProxyHandler<DeclarativeCallAgent> {
         };
       }
       case 'dispose': {
+        // Wrapping CallAgent.dispose in a callback type (): Promise<void> to accomodate the change of CallAgent.dispose
+        // in calling beta version 1.8.0-beta.1 from callback type (): Promise<void> to (): void
+        const disposeCallbackWrapper = (): Promise<void> => {
+          target.dispose();
```

Comment:
Can `target.dipose()` throw errors?

Reply:
I suppose it can. But we never had any catch logic before and an error in the console log will still show.